README.md

```diff
@@ -1,26 +1,77 @@
 # Loop Exercises
 
 1. Write a `while` loop and a `for` loop that takes the variable "num" and logs all the numbers, descending, between "num" and 1.
-2. Write a `while` loop and a `for` loop that takes the variable "num", and iterates over all numbers from 0 to "num". 
+``` js
+let num = 80;
+while num > 1 {
+  console.log(num);
+  num--;
+}
+for (let num = 80; num >1; num--){
+  console.log(num);
+}
+```
+2. Write a `while` loop and a `for` loop that takes the variable "num", and iterates over all numbers from 0 to "num".
 For each iteration, it will check if the current number is even or odd, and log that to the screen (e.g. "2 is even")
-3. Write a `while` loop and a `for` loop that takes the variable "num" and iterates over all numbers from 0 to "num". 
+``` js
+let num = 80;
+while (num >= 0) {
+  if (num%2===0) {
+  	console.log(num+" is even");
+  } else {
+  console.log(num+" is odd");
+  }
+num--
+}
+for (let num = 80; num>=0; num--) {
```

Comment:
Watch out for inconsistent spacing like this - this won't throw an error, but will give you a headache trying to find syntax errors if you ever get one!